### ISSUE TITLE
Add comment for label_smoothing parameter in torch.nn.CrossEntropyLoss 

### ISSUE THREAD

**Issue by asusdisciple** [pytorch/pytorch#129050]
### 🚀 The feature, motivation and pitch

There should be a hint or small warning for using `CrossEntropyLoss` with the parameter `label_smoothing`. Why you ask? I think for most use cases the user defines the same loss criterion for train and test/validation loops. Which makes perfectly sense!

However when you set the parameter `label_smoothing` you need an additional criterion on the test set, since you do not want to distort your target labels. I mean you would not smooth your labels during inference as well right?

However this can be quickly overseen at the moment. I can not think of a case where `label_smoothing `would be beneficial in a test/validation loop, so hard coding it would be an option, but would also add boiler code. I think documenting it in the docs would be fine, just writing a line like "If you use this parameter, define an additional loss criterion for your test loop, so that your true target labels are not distorted".

Maybe I also have a wrong thought here, correct me if I am wrong :)

Best regards

**Comment by jbschlosser** [comment]
My opinion is that there's enough uncertainty on whether label smoothing should be enabled during validation (perhaps considering it as a form of regularization) that the docs shouldn't take a strong stance either way (e.g. see discussion [here](https://datascience.stackexchange.com/questions/76494/should-you-turn-off-label-smoothing-when-validating)), but I'm open to being convinced otherwise.
